src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -24,6 +24,7 @@
 from nacl import encoding, public
 
 import OpenSSL.crypto
+import urllib3
```

Comment:
Do we need to add this library in requirements.txt somewhere?

Reply:
no, this is already in use today